client/www/pages/docs/explorer-component.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Explorer Component
+description: Use the Explorer Component in your own apps
+---
+
+The explorer page from the Instant Dashboard is available as an installable React component that you can use in your own apps!
+
+```tsx
+import './App.css';
+import { Explorer, Toaster } from '@instantdb/components';
+
+function App() {
+  return (
+    <>
+      <Explorer
+        className="h-full"
+        darkMode={false}
+        appId={import.meta.env.VITE_INSTANT_APP_ID}
+        adminToken={import.meta.env.VITE_INSTANT_ADMIN_TOKEN}
+      />
+      <Toaster position="top-right" />
+    </>
+  );
+}
+```
+
+The Toaster component is required for pop-up alerts, unless you are already using [Sonner](https://sonner.emilkowal.ski/) in your project.
+
+# CSS
+
+## Projects Using Tailwind
+
+For projects using Tailwind, add the component library to your Tailwind configuration.
+
+Tailwind V3:
+
+```js
+// tailwind.config.js
+module.exports = {
+  content: [
+    './lib/**/*.{js,ts,jsx,tsx}',
+    './pages/**/*.{js,ts,jsx,tsx,md}',
+    './_posts/**/*.{js,ts,jsx,tsx,md}',
+    './components/**/*.{js,ts,jsx,tsx}',
+    './utils/**/*.{js,ts,jsx,tsx}',
+    './node_modules/@instantdb/components/src/**/*.{js,ts,jsx,tsx,md}',
+  ],
+};
+```
+
+Tailwind V4:
+
+```css
+@import 'tailwindcss';
+@source '../node_modules/@instantdb/components';
+```
+
```

Comment:
Since useShadowDOM is true by default, we'd probably want to tell them to set false?